tests/static_analyzer/test_lsp_client.py

```
@@ -0,0 +1,615 @@
+        client = LSPClient(["cmd"], Path("/root"))
+        client._process = MagicMock()
+        client._process.poll.return_value = None
+
+        # Queue 3 identical errors
+        for i in range(1, 4):
+            client._msg_queue.put({"jsonrpc": "2.0", "id": i, "error": {"code": 0, "message": "no package metadata"}})
+
+        import logging
+
+        with caplog.at_level(logging.WARNING):
+            results, _, error_ids = client._collect_batch_responses([1, 2, 3], timeout=5)
```

Ruff (0.15.6)

[warning] Unpacked variable `results` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)